src/Utility/Unaligned.h

```diff
@@ -11,6 +11,6 @@
 template<class T>
 T readUnaligned(const void *data) {
     T result;
-    memcpy(&result, data, sizeof(T));
+    memcpy(&result, data, sizeof(T));  // TODO(yoctozepto): this works correctly only on little-endian systems
```

Comment:
This function is about avoiding UB, irrespective of whether we're reading serialized data or not. I would suggest to not even think about endianess, we're in 2024 X)